test/tests/functions/coarsened_piecewise_linear/tests

```diff
@@ -0,0 +1,17 @@
+[Tests]
+  issues = '#2272'
+  design = 'source/functions/CoarsenedPiecewiseLinear.md'
+
+  [./prepare_data]
+    type = RunCommand
+    command = './generate_data.py'
+    requirement = "Generate the fine tabulated function data for the coarsened_piecewise_linear test"
```

Comment:
Add the following, so this test is properly skipped, if numpy is not available:

```pre
required_python_packages = 'numpy'
```

Reply:
Does it make sense to just modify the script to not use numpy if that's an issue? Do we have systems where numpy isn't available?

Reply:
I don;t think there are systems where MOOSE is available, but it is impossible to get numpy. The test boxes certainly have it and my priority is to make those run as efficiently as possible.